private/bufpkg/bufimage/bufimage.go

```diff
@@ -31,19 +31,14 @@ import (
 type ImageFile interface {
 	bufmoduleref.FileInfo
 
-	// Proto is the backing *descriptorpb.FileDescriptorProto for this File.
+	// FileDescriptorProto is the backing *descriptorpb.FileDescriptorProto for this File.
 	//
 	// FileDescriptor should be preferred to Proto. We keep this method around
```

Comment:
nit: Assuming this comment needs to be updated?

Reply:
fixed